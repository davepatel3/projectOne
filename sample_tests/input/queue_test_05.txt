int
4
4
3
eb 23
ef 23
ef -47
eb -88
eb 35
df
gb
ef -17
db
df
ef -82
ps
ef -67
gf
eb 24
gb
ef -95
eb -99
pb
db
gs
ef -15
ef 60
db
pf
pf
df
ps
gc
eb 33
ps
ps
ps
eb -81
ef -11
ef -20
ef 90
gs
gf
ps
qt
