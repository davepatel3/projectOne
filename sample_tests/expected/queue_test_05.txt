-47
35
35
-17
[+][+][+][-][+]
-67
24
24
6
-88
-95
[+][+][+][+][-][-]
6
[+][+][+][+][+][-]
[+][+][+][+][+][-]
[+][+][+][+][+][-]
9
90
[+][+][+][+][+][+][-][-][-][-][-][-][-][-][-][-][-][-][-][-][-][+][+][+]
